engine/gamesys/src/gamesys/components/comp_spine_model.cpp

```diff
@@ -710,9 +710,10 @@ namespace dmGameSystem
             }
             else if (params.m_Message->m_Id == dmGameSystemDDF::SetConstantSpineModel::m_DDFDescriptor->m_NameHash)
             {
+                int index = 0; // TODO
```

Comment:
TODO what?

Reply:
I think we can probably wait with supporting arrays for some of these components, so I'll just remove the comment.